package.json

```diff
@@ -50,7 +50,9 @@
     "node-mocks-http": "1.2.3",
     "sinon": "1.17.6",
     "typescript": "2.0.10",
-    "typings": "2.0.0"
+    "@types/mocha": "^2.2.40",
+    "@types/node": "^7.0.12",
```

Comment:
In particular, the previous node typings was for node 0.10, but it looks like the earliest that the `@types/node` package has is for node 4.X